### ISSUE TITLE
Multiple (redundant) calls to lookup for the same input/output

### ISSUE THREAD

**Issue by slisznia** [Yubico/java-webauthn-server#155]
I've just implemented the authn with ver 1.11 of the library. It works. However the server logs indicate repeated calls (five in my case) to CredentialRepository::lookup implementation per single finishAssertion invocation. Everything from the input args to the results is identical between these successive calls.

```
INFO WebAuthn - lookup credential ID: ByteArray(0e17becba1612e7762...), user handle: ByteArray(4c415443484...); result: Optional[usr=ABC;tm=2021-11-23T14:36:33.920Z;cr=RegisteredCredential(credentialId=ByteArray(0e17be...046), userHandle=ByteArray(4c415443484...), publicKeyCose=ByteArray(a5032601022...), signatureCount=0)]
INFO WebAuthn - lookup credential ID: ByteArray(0e17becba1612e7762...), user handle: ByteArray(4c415443484...); result: Optional[usr=ABC;tm=2021-11-23T14:36:33.920Z;cr=RegisteredCredential(credentialId=ByteArray(0e17be...046), userHandle=ByteArray(4c415443484...), publicKeyCose=ByteArray(a5032601022...), signatureCount=0)]
INFO WebAuthn - lookup credential ID: ByteArray(0e17becba1612e7762...), user handle: ByteArray(4c415443484...); result: Optional[usr=ABC;tm=2021-11-23T14:36:33.920Z;cr=RegisteredCredential(credentialId=ByteArray(0e17be...046), userHandle=ByteArray(4c415443484...), publicKeyCose=ByteArray(a5032601022...), signatureCount=0)]
INFO WebAuthn - lookup credential ID: ByteArray(0e17becba1612e7762...), user handle: ByteArray(4c415443484...); result: Optional[usr=ABC;tm=2021-11-23T14:36:33.920Z;cr=RegisteredCredential(credentialId=ByteArray(0e17be...046), userHandle=ByteArray(4c415443484...), publicKeyCose=ByteArray(a5032601022...), signatureCount=0)]
INFO WebAuthn - lookup credential ID: ByteArray(0e17becba1612e7762...), user handle: ByteArray(4c415443484...); result: Optional[usr=ABC;tm=2021-11-23T14:36:33.920Z;cr=RegisteredCredential(credentialId=ByteArray(0e17be...046), userHandle=ByteArray(4c415443484...), publicKeyCose=ByteArray(a5032601022...), signatureCount=0)]
```

**Comment by emlun** [comment]
Yeah, I've kind-of known this for a while but never got around to fixing it. Thanks for the poke! This is now fixed in [pre-release `1.12.1-RC1`](https://github.com/Yubico/java-webauthn-server/releases/tag/1.12.1-RC1), available now on Maven Central. If nothing else comes up, we'll promote that to a proper `1.12.1` release in about two weeks.

**Comment by emlun** [comment]
The proper `1.12.1` release with the fix is now out on Maven Central. Thanks again!
